pysat/tests/classes/cls_instrument_library.py

```diff
@@ -0,0 +1,404 @@
+"""Standardized class and functions to test instruments for pysat libraries.
+
+Note
+----
+Not directly called by pytest, but imported as part of test_instruments.py.
+Can be imported directly for external instrument libraries of pysat instruments.
+
+Examples
+--------
+::
+
+    # Import custom instrument library
+    import mypackage
+
+    # Import the test classes from pysat.
+    from pysat.tests.classes.cls_instrument_library import InstLibTests
+
+    InstLibTests.initialize_test_package(InstLibTests,
+                                         inst_loc=mypackage.instruments,
+                                         user_info=user_info)
+
+    class TestInstruments(InstLibTests):
+        '''Create a testable object from standard library.'''
```

Comment:
```suggestion
        '''Create a testable object from standard library.
        
        Note
        -----
        In your docstring be sure to use double quotes instead of single quotes.
        
        '''
```